From ca9eb5bdbdfe17654466d84c8baaa1187a8796c6 Mon Sep 17 00:00:00 2001
From: Yong Gan <b45748@freescale.com>
Date: Wed, 2 Jul 2014 11:27:26 +0800
Subject: [PATCH] ENGR00320243 Enable GAL2D compositor in SoloLite
Organization: O.S. Systems Software LTDA.

Build gal2d-renderer.so when EGL was not enabled.

Date: Jul 02, 2014
Signed-off-by Yong Gan <yong.gan@freescale.com>

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 29834c3..e82e970 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -165,6 +165,7 @@ gl_renderer_la_SOURCES =			\
 	src/gl-renderer.c			\
 	src/vertex-clipping.c			\
 	src/vertex-clipping.h
+endif
 
 module_LTLIBRARIES += gal2d-renderer.la
 gal2d_renderer_la_LDFLAGS = -module -avoid-version
@@ -178,7 +179,6 @@ gal2d_renderer_la_SOURCES =			\
 	src/gal2d-renderer.c			\
 	src/vertex-clipping.c			\
 	src/vertex-clipping.h
-endif
 
 if ENABLE_X11_COMPOSITOR
 module_LTLIBRARIES += x11-backend.la
-- 
2.1.4

